From 1923c97fbd0b3d8cacdb72386886f860c818bccf Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Wed, 10 Nov 2021 08:54:06 -0700
Subject: [PATCH] meson: remove session-managers option check.

This option no longer exists and causes a build failure if building
with the systemd system service enabled.

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[Upstream status:
https://gitlab.freedesktop.org/pipewire/media-session/-/merge_requests/24]
---
 systemd/system/meson.build | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/systemd/system/meson.build b/systemd/system/meson.build
index ab0d49064..0e9885b0f 100644
--- a/systemd/system/meson.build
+++ b/systemd/system/meson.build
@@ -4,9 +4,7 @@ systemd_system_services_dir = systemd.get_variable(pkgconfig: 'systemdsystemunit
 systemd_config = configuration_data()
 systemd_config.set('PW_MEDIA_SESSION_BINARY', media_session_bindir / 'pipewire-media-session')
 
-if get_option('session-managers').contains('media-session')
-  configure_file(input : 'pipewire-media-session.service.in',
-               output : 'pipewire-media-session.service',
-               configuration : systemd_config,
-               install_dir : systemd_system_services_dir)
-endif
+configure_file(input : 'pipewire-media-session.service.in',
+             output : 'pipewire-media-session.service',
+             configuration : systemd_config,
+             install_dir : systemd_system_services_dir)
-- 
2.25.1

